vagrant/UPGRADING.md

```diff
@@ -0,0 +1,49 @@
+UPGRADING
+=========
+
+0.1.0
+-----
+This adds the possibility to have a whole hiera hierarchy.
```

Comment:
thats the magic behind this PR. [Hiera](http://docs.puppetlabs.com/hiera/1/) is a kind of sorted data or parameters for puppet, so you have an encapsulation of parameters and "logic" to build stuff.